rust/src/writer/record_batch.rs

```diff
@@ -410,6 +403,18 @@ pub(crate) fn divide_by_partition_values(
     Ok(partitions)
 }
 
+fn lexsort_to_indices(arrays: &[ArrayRef]) -> UInt32Array {
```

Comment:
I am not following why this function is being used here instead of the previous import from `arrow::compute` as far as I can tell the only difference is this version doesn't take the optional `limit` parameter?

:confused:

Reply:
This version uses arrow-row rather then the "regular" compute kernels. https://github.com/apache/arrow-rs/blob/77aa8f5b2645a91724048f5c1d644c6b52880028/arrow-ord/src/sort.rs#L1081-L1082